scripts/run_mypy_checks.py

```diff
@@ -197,7 +197,21 @@
     'core/tests/load_tests/feedback_thread_summaries_test.py',
     'core/tests/test_utils.py',
     'core/tests/test_utils_test.py',
-    'core/jobs',
+    'core/jobs/batch_jobs',
+    'core/jobs/decorators',
+    'core/jobs/transforms',
```

Comment:
Should also the contents of these be added instead of the whole folders?

Reply:
For simplicity, I think we should not add the contents of these folders until someone wants to work on them. Because it only increases the listed size if no one working on it for now.

Reply:
Ok